incubator/couchbase/templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+########################################################################
+Couchbase can be accessed via port 8091 when the service will be created
+
+Default Username : Administrator / Password : password
```

Comment:
Can you add port forwarding instructions here. A good example is the Jenkins NOTES.txt:

https://github.com/kubernetes/charts/blob/master/stable/jenkins/templates/NOTES.txt